client/src/components/events-components/EventCard.tsx

```diff
@@ -1,11 +1,16 @@
 // Need to define an interface for an event
 interface EventProps {
-  event: object;
+  event: Event;
 }
+
+interface Event {
+  name: String;
+}
+
 const EventCard = ({ event }: EventProps) => {
   return (
     <article className="flex flex-col justify-start itmes-center w-75 h-100 m-3 p-1.5 border rounded-xl">
```

Comment:
you can use the Card component:
https://ui.shadcn.com/docs/components/card

for future UI implementation, make sure go through the shadcn components page to see if there are any can be reused:
https://ui.shadcn.com/docs/components